client/app/templates/projects.hbs

```diff
@@ -0,0 +1,61 @@
+<div class="site-main large-padding-bottom">
+    <section class="grid-container large-padding-top large-padding-bottom">
+
+    <div class="grid-x grid-padding-x">
+        {{!-- breadcrumb --}}
+        <div class="cell">
+            <nav aria-label="You are here:" role="navigation">
+                <ul class="breadcrumbs">
+                <li><span class="show-for-sr">Current: </span> My Projects</li>
+                </ul>
+            </nav>
+            <hr />
+        </div>
+
+        {{!-- left list column --}}
+        <div class="cell auto">
+            {{!-- todo projects list header --}}
+            <h4 class="text-dark-gray large-margin-bottom text-weight-normal">
+                TO DO
+            </h4>
+            {{!-- projects list --}}
+            <ul class="no-bullet">
+                {{#each this.applicantProjects as |project|}}
+                    <Projects::ApplicantProjectCard @responsibility="applicant" @project={{project}} />
```

Comment:
we should nix the `@ responsibility` parameter if it's not getting used

Reply:
@bfreeds you wanna address these?

Reply:
@godfreyyeung  @allthesignals am working on it

Reply:
resolved in #52